pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/redigerbar/BekreftelsePaaFlyktningstatus.kt

```diff
@@ -0,0 +1,71 @@
+package no.nav.pensjon.brev.maler.redigerbar
+
+import no.nav.pensjon.brev.api.model.TemplateDescription
+import no.nav.pensjon.brev.api.model.maler.Brevkode
+import no.nav.pensjon.brev.api.model.maler.EmptyRedigerbarBrevdata
+import no.nav.pensjon.brev.maler.fraser.common.Felles
+import no.nav.pensjon.brev.maler.fraser.common.Felles.fulltNavn
+import no.nav.pensjon.brev.template.Language.Bokmal
+import no.nav.pensjon.brev.template.RedigerbarTemplate
+import no.nav.pensjon.brev.template.dsl.createTemplate
+import no.nav.pensjon.brev.template.dsl.expression.plus
+import no.nav.pensjon.brev.template.dsl.helpers.TemplateModelHelpers
+import no.nav.pensjon.brev.template.dsl.languages
+import no.nav.pensjon.brev.template.dsl.text
+import no.nav.pensjon.brev.template.dsl.textExpr
+import no.nav.pensjon.brevbaker.api.model.BrukerSelectors.foedselsnummer
+import no.nav.pensjon.brevbaker.api.model.FellesSelectors.avsenderEnhet
+import no.nav.pensjon.brevbaker.api.model.FellesSelectors.bruker
+import no.nav.pensjon.brevbaker.api.model.FoedselsnummerSelectors.value
+import no.nav.pensjon.brevbaker.api.model.LetterMetadata
+import no.nav.pensjon.brevbaker.api.model.NAVEnhetSelectors.navn
+
+
+@TemplateModelHelpers
+object BekreftelsePaaFlyktningstatus : RedigerbarTemplate<EmptyRedigerbarBrevdata> {
+
+    override val kode = Brevkode.Redigerbar.PE_BEKREFTELSE_PAA_FLYKTNINGSTATUS
+    override val kategori: TemplateDescription.Brevkategori =
+        TemplateDescription.Brevkategori.INNHENTE_OPPLYSNINGER
+
+
+    override val template = createTemplate(
+        name = kode.name,
+        letterDataType = EmptyRedigerbarBrevdata::class,
+        languages = languages(Bokmal),
+        letterMetadata = LetterMetadata(
+            displayTitle = "Bekreftelse på flyktningstatus",
+            isSensitiv = false,
+            distribusjonstype = LetterMetadata.Distribusjonstype.VIKTIG,
+            brevtype = LetterMetadata.Brevtype.INFORMASJONSBREV
+        )
+    ) {
+        title {
+            text(
+                Bokmal to "Bekreftelse på flyktningstatus"
+            )
+        }
+        outline {
+            paragraph {
```

Comment:
Skal det være så mange paragraphs?

Reply:
Eller kanskje ikke :sweat_smile:. Vi har ikke støtte for newline i skribenten. Grunnen til at vi innførte  `newline` var egentlig bare for å støtte noen gamle brev som hadde "rar" formattering. Så derfor har vi ikke støtte for det i skribenten.